docs/troubleshooting.adoc

```diff
@@ -0,0 +1,157 @@
+= Troubleshooting Landrush
+:toc:
+:toc-placement!:
+
+Use this document to identify and resolve basic Landrush issues that you encounter.
+https://github.com/vagrant-landrush/landrush#landrush-dns-for-vagrant-[Landrush]
```

Comment:
This is probably not the best link to reference. Why do you not point to the README directly or alternatively leave of the anchor in the link above.